### DISCUSSION TITLE
adding a Drawer component

### DISCUSSION THREAD

**Discussion by mohammad-reza-tajik** [tailwindlabs/headlessui#2797] [Ideas]
Hi , thanks for your amazing project ,
I have a simple question.  Is there any possibility that you add a Drawer component ?. that would be amazing!

**Comment by ChristofferWolgast** [comment]
Just use a Dialog with the proper transition that you're after.

**Reply by Emiliano-Bucci** [reply]
@ChristofferWolgast Yes but if you need to render a Drawer SSR this won't work, even using the static prop :/
